.github/workflows/Crane.yaml

```diff
@@ -2,8 +2,6 @@ name: Crane
 
 on:
   push:
-    branches:
```

Comment:
Why the removal? (ditto other projects)

Reply:
So that it builds this branch. This is pulled in from @JoseAlcerreca 's [change](https://github.com/android/compose-samples/commit/5f9e1a83007a86680e37bafb26a970fce1c36b29).

Reply:
You don't need that, the `pr` block means that it will build as soon as you create a PR (and update the PR).

I added this to all of my projects because I find that having actions run on draft branches annoying, and wasteful.